src/inc/quic_trace.h

```diff
@@ -123,9 +123,1124 @@ QuicTraceRundown(
 
 #ifdef QUIC_EVENTS_STUB
 
+#define QUIC_TRACE_EVENT(x, ...) 
```

Comment:
@maximus- why all this?

Reply:
I didn't have a way to enable the tracing on macOS so I ended up forwarding this all to a printf call and this was just part of the hack. I could've sworn I removed it...